### ISSUE TITLE
fastq_stats: corner case when computing truncation percentage

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/vsearch#355]
With the this toy-example containing only one sequence of one nucleotide:
```sh
printf "@s\nA\n+\nG\n" | vsearch --fastq_stats - --log -
```
the log file contains:
```
vsearch v2.10.2_linux_x86_64, 62.8GB RAM, 8 cores
https://github.com/torognes/vsearch

Reading FASTQ file 100%

Read length distribution
      L           N      Pct   AccPct
-------  ----------  -------  -------
>=    1           1   100.0%   100.0%

Q score distribution
ASCII    Q       Pe           N      Pct   AccPct
-----  ---  -------  ----------  -------  -------
    G   38  0.00016           1   100.0%   100.0%

    L  PctRecs  AvgQ  P(AvgQ)      AvgP  AvgEE       Rate   RatePct
-----  -------  ----  -------  --------  -----  ---------  --------

    L   1.0000   0.5000   0.2500   0.1000   1.0000   0.5000   0.2500   0.1000
-----  -------  -------  -------  -------  -------  -------  -------  -------
    1        1        1        1        1  100.00%  100.00%  100.00%  100.00%

Truncate at first Q
  Len     Q=5    Q=10    Q=15    Q=20
-----  ------  ------  ------  ------
    1  100.0%  100.0%  100.0%  100.0%
    0    0.0%    0.0%    0.0%  1640100.0%

         1  Recs (0.0M), 0 too long
       1.0  Avg length
      0.0M  Bases
Read 1 sequences.
```
Notice the `    0    0.0%    0.0%    0.0%  1640100.0%`.

When testing with a sequence of two nucleotides, `vsearch` produces the expected output:
```
Truncate at first Q
  Len     Q=5    Q=10    Q=15    Q=20
-----  ------  ------  ------  ------
    2  100.0%  100.0%  100.0%  100.0%
    1  100.0%  100.0%  100.0%  100.0%
```

**Comment by torognes** [comment]
Fixed in commit a32173e.

**Comment by frederic-mahe** [comment]
I've just added a test for that specific issue.

**Comment by torognes** [comment]
Fixed in version 2.10.4, just released.
